checker/return_error_last_checker.go

```diff
@@ -18,6 +18,26 @@ func NewReturnErrorLastChecker(configData interface{}) NodeChecker {
 	return &ReturnErrorLastChecker{}
 }
 
+// Title implements the NodeChecker interface.
+func (c *ReturnErrorLastChecker) Title() string {
+	return "Error Return Last"
+}
+
+// Description implements the NodeChecker interface.
+func (c *ReturnErrorLastChecker) Description() string {
+	return `A function must return error as its last argument.`
```

Comment:
Similar to the `context.Context` checker, I suggest `When a function returns error it must be its last return value.`.

Reply:
Done.